### ISSUE TITLE
[Bug]: KAT Advanced Medical items don't show up in Arsenal

### ISSUE THREAD

**Issue by Crazys-Corner** [official-antistasi-community/A3-Antistasi#2481]
### Describe the bug

KAT Advanced medical (which builds upon ace's medical system) has certain items to treat patients. These items are not unlocked by default. I ended up Zeusing them in (25 of each) and they did not show up in the Arsenal.

### How to reproduce

- Login as Admin (If it's a Dedicated Server) 
- Shift + Y to enable Zeus
- Spawn in any object and put an Arsenal on it
- Spawn in 25 of any Kat Medical items
- Put them in the Inventory spot of the main arsenal
- open the main arsenal and click "To Crate"
- Check and see if it gives the items (it doesn't)

### Version

2.5.5

### Have you altered the code?

No

### Map

Altis

### What server?

Private dedicated server

### Mods

```html
@cba_a3;@ace;@rhsusaf;@rhssaf;@rhsgref;@rhsafrf;@acecompat-rhs-afrf;@acecompat-rhs-saf;@acecompat-rhs-gref;@acecompat-rhs-usaf;@projectopfor;@cupterrains-core;@cupterrains-maps;@cupterrains-cwa;@cupterrains-core;@uspgear&uniformsaio;@usafmod-utility;@usafmod-main;@usafmodfighters;@tfaranimations;@taskforcearrowheadradio(beta!!!);@fallujah;@enhancedmovement;@enhancedmovementrework;@duisquadradar;@zeusenhanced;@zeusadditions;@zeusimmersionsounds;@zeusenhancedace3compatibility;@align;@advancedweaponmounting;@advancedurbanrappelling;@advancedslingloading;@advancedrappellingreworked;@advanceaeroeffects;@taskforcearrowheadradio(beta!!!);@nomoreaircraftbouncing;@lambs_turrets;@lambs_supression;@lambs_rpg;@lambs_danger.fsm;@kka3injuredanimations;@katadvancedmedicalrewrite;@kamrhscompatibility;@immersivehalojumpmod[usaf+rhs];@backpackonchestredux;@remoteedsseperatedetonation;@firesupportplus;
```

**Comment by killerswin2** [comment]
Kat medical is supported in the next release. Until that release kat med will be broken unfortunately.

**Comment by jaj22** [comment]
@killerswin2 That KAT patch didn't touch the arsenal code though. Did you check whether the items actually show up there?

**Comment by killerswin2** [comment]
I did, the items where added to initial equipment

**Comment by killerswin2** [comment]
However that was over a year ago

**Comment by Bob-Murphy** [comment]
Thank you for the issue.
1. Which KAT version are you using? There are two. The old and discontinued one with only 7k subs or the rewrite with 62k subs? Please always make sure to directly reference the steam workshop item when talking about a "new" 3rd party mod.
2. I'm in contact with one of the creators of the rewrite and we will overhaul out integration once they have released the next update.

**Comment by Crazys-Corner** [comment]
I'm using KAT Advanced Medical REWRITE

Also I obviously have all compatibility mods

**Comment by killerswin2** [comment]
fixed in #2523
